tests/test_inverse_collation.py

```diff
@@ -53,6 +53,19 @@
     ]
 ]
 
+TESTS_2D = [
+    (t.__class__.__name__ + (" pad_list_data_collate" if collate_fn else " default_collate"), t, collate_fn, 2)
+    for collate_fn in [None, pad_list_data_collate]
+    for t in [
+        RandFlipd(keys=KEYS, spatial_axis=[1]),
+        RandAxisFlipd(keys=KEYS),
+        RandRotate90d(keys=KEYS, spatial_axes=(0, 1)),
+        RandZoomd(keys=KEYS, prob=0.5, min_zoom=0.5, max_zoom=1.1, keep_size=True),
+        RandRotated(keys=KEYS, range_x=np.pi),
```

Comment:
By default probability is 0.1, so for batch size of 7, there's no guarantee of a mix of `True` and `False` for `_do_transform`. Should probably just set `prob=0.5` for all transforms.

Reply:
good point, I'll revise...